terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/resources/org/terasoluna/gfw/common/codelist/i18n/simpleI18nCodeList.xml

```diff
@@ -147,13 +189,39 @@
             <entry key="6" value="土" />
           </util:map>
         </entry>
+        <entry key="ja_JP">
+          <util:map>
+            <entry key="0" value="日" />
+            <entry key="1" value="月" />
+            <entry key="2" value="火" />
+            <entry key="3" value="水" />
+            <entry key="4" value="木" />
+            <entry key="5" value="金" />
+            <entry key="6" value="土" />
+          </util:map>
+        </entry>
       </util:map>
     </property>
-    <property name="rowsByCodeList">
+    <property name="fallbackTo" value="ja_JP" />
+  </bean>
+
+  <bean id="CL_testResolveLocaleFallbackToUS" class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList">
+    <property name="rows">
       <util:map>
-        <entry key="en" value-ref="CL_WEEK_EN" />
-        <entry key="ja" value-ref="CL_WEEK_JA" />
+        <entry key="en_US">
+          <util:map>
+            <entry key="0" value="Sun." />
+            <entry key="1" value="Mon." />
+            <entry key="2" value="Tue." />
+            <entry key="3" value="Wed." />
+            <entry key="4" value="Thu." />
+            <entry key="5" value="Fri." />
+            <entry key="6" value="Sat." />
+          </util:map>
+        </entry>
       </util:map>
     </property>
+    <property name="fallbackTo" value="en_US" />
   </bean>
```

Comment:
Modified by 2f5f06aa491b3d8311ac129f4a69a708f6207d3f

There was no Swiss French (fr-ch) locale in the locale class.
Only those that exist in the locale class can be defined in the `entry` tag.
Therefore, it was replac as follows.

* Bean
```
id=CL_testResolveLocale
  property=rows
    entry=de
    entry=fr
    entry=fr-ca
    entry=ja
    entry=en-us
  fallbackTo=de  <-  Since it became necessary, setting
```
* JUnit
```
asMap("fr") -> return "fr" specific language locale.
asMap("fr-ca") -> return "fr-ch" specific nation locale.
asMap("ja") -> return "ja" specific language locale. (Not implemented)
asMap("ja_JP") -> return "ja" resolved language locale.
asMap("en") -> return "de" fallback locale.
asMap("en-us") -> return "at" specific nation locale. (Not implemented)
asMap("en-ca") -> return "en" fallback locale.
```